src/dashboard.tsx

```diff
@@ -609,6 +616,37 @@ namespace Private {
         });
     }
 
+    else if (enableDashboardFetch) {
```

Comment:
Style nit:
```suggestion
    } else if (enableDashboardFetch) {
```


Note that there are some pre-commit hooks that you can install to run linter/formatting checks automatically. But I can also run them in a separate commit later if you're having trouble with them.

Reply:
I ran pre-commit this time, hope these errors disappear :slightly_smiling_face: